platform/ios/src/MGLMapView.mm

```diff
@@ -122,6 +122,9 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingState) {
 
 const CGSize MGLAnnotationAccessibilityElementMinimumSize = CGSizeMake(10, 10);
 
+// Context for KVO observing UILayoutGuides.
+static char MGLLayoutGuidesUpdatedContext = 1;
```

Comment:
This works, but the more common practice is to pass in a selector such as `_cmd`.